parachain/runtime/src/lib.rs

```diff
@@ -333,11 +335,19 @@ impl xcm_handler::Config for Runtime {
 
 // Our pallets
 
+parameter_types! {
+	pub Apps = {
+		let mut registry = make_registry();
+		registry.insert(H160::repeat_byte(1), eth_app::make_proxy<Runtime>())
```

Comment:
Still need to settle on a suitable type for application IDs (Still using `H160` here), so channels can dispatch to them.

Apps still need to know the contract address of their peers on the Ethereum side though. That code hasn't been removed.